internal/acctest/acctest.go

```diff
@@ -59,22 +66,28 @@ func TestClient() *client.APIClient {
 			URL: httpsHost + adminApiPath,
 		},
 	}
-	// Trusting all for the acceptance tests, since they run on localhost
-	// May want to incorporate actual trust here in the future.
-	//#nosec G402
-	tr := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-	}
-	httpClient := &http.Client{Transport: tr}
+
+	httpClient := &http.Client{Transport: GetTransport()}
 	clientConfig.HTTPClient = httpClient
 	return client.NewAPIClient(clientConfig)
 }
 
+// lintignore:AT008
+func TestAccessTokenContext(accessToken string) context.Context {
+	ctx := context.Background()
+	return config.AccessTokenContext(ctx, accessToken)
+}
+
 func TestBasicAuthContext() context.Context {
 	ctx := context.Background()
 	return config.BasicAuthContext(ctx, os.Getenv("PINGFEDERATE_PROVIDER_USERNAME"), os.Getenv("PINGFEDERATE_PROVIDER_PASSWORD"))
 }
 
+func TestOauth2Context() context.Context {
+	ctx := context.Background()
+	return config.OAuthContext(ctx, GetTransport(), os.Getenv("PINGFEDERATE_PROVIDER_OAUTH_TOKEN_URL"), os.Getenv("PINGFEDERATE_PROVIDER_OAUTH_CLIENT_ID"), os.Getenv("PINGFEDERATE_PROVIDER_OAUTH_CLIENT_SECRET"), []string{os.Getenv("PINGFEDERATE_PROVIDER_OAUTH_SCOPES")})
```

Comment:
It'd be good to have some error reporting here if any of these env vars aren't set.